techlibs/sf2/synth_sf2.cc

```diff
@@ -209,10 +209,14 @@ struct SynthSf2Pass : public ScriptPass
 
 		if (check_label("map_iobs"))
 		{
-			if (help_mode)
-				run("sf2_iobs [-clkbuf]", "(unless -noiobs)");
-			else if (iobs)
-				run(clkbuf ? "sf2_iobs -clkbuf" : "sf2_iobs");
+			if (help_mode || iobs) {
+				if (help_mode || clkbuf) {
+					run("clkbufmap -buf $_BUF_ Y:A -inpad CLKBUF Y:PAD", "(if -clkbuf, unless -noiobs)");
```

Comment:
now that #2230 has landed, these three lines can be replaced by `clkbufmap -buf CLKINT Y:A -inpad CLKBUF Y:PAD`